tools/ci_build/github/azure-pipelines/android-x86_64-crosscompile-ci-pipeline.yml

```diff
@@ -23,7 +23,26 @@ jobs:
         --emulator-pid-file $(Build.BinariesDirectory)/emulator.pid
     displayName: Start Android emulator
 
-  - script: python3 tools/ci_build/build.py --android --build_dir build --android_sdk_path $ANDROID_HOME --android_ndk_path $ANDROID_HOME/ndk-bundle --android_abi=x86_64 --android_api=29 --skip_submodule_sync --parallel --cmake_generator=Ninja --build_java
+  # Start switching to jdk 11 after the Android Emulator is started since Android SDK manager requires java 8
+  - task: JavaToolInstaller@0
```

Comment:
>task: JavaToolInstaller@0 [](start = 4, length = 25)

Will this execute once?

Reply:
Yes, this task is actually setting the environment variables